e2e_tests/test_airlock.py

```diff
@@ -73,10 +73,10 @@ async def submit_airlock_import_request(workspace_path: str, workspace_owner_tok
 
 @pytest.mark.timeout(50 * 60)
 @pytest.mark.airlock
-async def test_airlock_review_vm_flow(setup_test_workspace_and_workspace_service, setup_test_airlock_import_review_workspace_and_workspace_service, verify):
-    workspace_path, workspace_id, workspace_service_path, workspace_service_id, workspace_owner_token = setup_test_workspace_and_workspace_service
-    _, import_review_workspace_id, _, import_review_workspace_service_id, _ = setup_test_airlock_import_review_workspace_and_workspace_service
-    LOGGER.info("Workspace and workspace service set up")
+async def test_airlock_review_vm_flow(setup_test_workspace, setup_test_airlock_import_review_workspace_and_guacamole_service, verify):
+    workspace_path, workspace_id, workspace_owner_token = setup_test_workspace
+    _, import_review_workspace_id, _, import_review_workspace_service_id, _ = setup_test_airlock_import_review_workspace_and_guacamole_service
+    LOGGER.info("Base workspace set up")
```

Comment:
```suggestion
    LOGGER.info("Import review workspace & Guacamole setup is done.")
```

Reply:
at this point both workspaces are ready, shouldn't we mention both of them? 
`Workspaces setup is done.`

Or the elaborated one:

`Base workspace and Review workapace with Guacamole are set up`

Reply:
isn't the fixture(s) anyway log that it did its work?

Reply:
yes, they do, so we can remove it,
but on the other hand, it's sometimes convenient to look for a log that was generated by the test itself, to know where the real test starts

Reply:
removed